Automattic/super-cool-ad-inserter-plugin
# Update block editor style to look like a `Placeholder`

At the moment, the editor stylesheet doesn't load. This PR fixes it. AND style the block like a Gutenberg `Placeholder`.

__How to test__

1. On a page/post/CPT, add the block
2. Notice how unstyled it is (and check the sidebar -- note: why do we need the setting there as well??)
3. Switch to this branch
4. 🧙‍♂️

Closes #44
